pkg/cn-cbor/Makefile

```diff
@@ -7,6 +7,7 @@ include $(RIOTBASE)/pkg/pkg.mk
 
 # Enable code forcing aligned reads
 CFLAGS += -DCBOR_ALIGN_READS
+CFLAGS += -Wno-return-local-addr
```

Comment:
Do you think this one can be fixed upstream?

Reply:
Maybe but I don't know that code enough.

Reply:
Ok.